coordinator/src/settings.rs

```diff
@@ -15,6 +15,9 @@ const SETTINGS_FILE_NAME: &str = "coordinator-settings.toml";
 /// Top-level settings.
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct Settings {
+    /// If set to true, new users are allow to trade on the platform without
+    /// previously being on the waitlist.
+    pub auto_allow_new_users: bool,
```

Comment:
Should we add a cli arg to overwrite this? Otherwise it might get cumbersome with testing.